bin/gwdetchar-lasso-correlation

```diff
@@ -128,7 +124,7 @@ psig.add_argument('-x', '--filter-padding', type=float, default=3.,
 lsig = parser.add_argument_group('LASSO options')
 lsig.add_argument('-a', '--alpha', default=None, type=float,
                   help='alpha parameter for LASSO fit')
-lsig.add_argument('-C', '--cluster', default=True, type=bool,
+lsig.add_argument('-C', '--cluster', default='True', type=str,
```

Comment:
I don't understand this change, can you explain?

Reply:
@duncanmmacleod As I discovered from troubleshooting and reading more online, for some reason, boolean arguments will always be `True` whether you pass in `True` or `False`. There are a few different ways to work around this issue, but I found this the simplest and most readable.

Reply:
I believe it has something to do with the nature of python or parsing. It will treat any string passed as a boolean as `True`

Reply:
in this case you want to use [`action='store_true'`](https://docs.python.org/2/library/argparse.html#action) and just give the argument with a value, e.g.

```python
lsig.add_argument('-C', '--cluster', action='store_true', 
                  help='generate clustered channel plots')
```

the user should then pass just `--cluster` to enable the clustering.

Reply:
@duncanmmacleod I'm happy to change this, but our overall usage preference has been to always do clustering by default, and only turn it off if in special cases (e.g. when troubleshooting the code overall to fix bugs). If we would like to keep it as doing clustering by default without the user passing anything, is there another way you would like me to rewrite it? If you still feel this new way you proposed is best, I will make the switch.

Reply:
@macedo22, you can use the opposite logic and provide something like:

```python
lsig.add_argument('-C', '--no-cluster', action='store_true', default=False,
                  help='do not generate clustered channel plots')
```

I believe that one of those options using `action='store_true'` is better than having the user manually pass `true` or `false` on the command-line.

Reply:
@duncanmmacleod Sounds good to me. I swapped this in and tested it, so it should be good to go with this most recent commit `56d11d5`.